Pod/Classes/CZPickerView.m

```diff
@@ -308,7 +308,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             cell.accessoryType = UITableViewCellAccessoryCheckmark;
         }
     }
-    if ([self.dataSource respondsToSelector:@selector(czpickerView:attributedTitleForRow:)]) {
+    if([self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)] && [self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)]){
```

Comment:
True, therefore you should check it ```dataSource``` responds to ```czpickerView:titleForRow:``` AND ```czpickerView:imageForRow:``` right?

You are actually checking ```czpickerView:titleForRow:``` twice.